setup.py

```diff
@@ -13,6 +13,14 @@
     'pyexcel-webio>=0.0.3',
     'Flask>=0.10.1'
 ]
+extras = {
+    'xls': ['pyexcel-xls==0.0.7'],
+    'xlsx': ['pyexcel-xlsx==0.0.7'],
+    'ods3': [
+        'pyexcel-ods3==0.0.8',
+        'git+https://github.com/T0ha/ezodf.git@2c69103e6c0715adb0e36562cb2e6325fd776112#egg=ezodf-master',
```

Comment:
The same comment as the one in pyexcel-io. Shall we use zip file instead of cloning it? Let's have zip and continue the merge